packages/shared/lib/services/sync/config/deploy.service.ts

```diff
@@ -91,12 +97,19 @@ export async function deploy({
 
         idsToMarkAsInactive.push(...response.idsToMarkAsInactive);
         syncConfigs.push(response.syncConfig);
-        deployResults.push({
+        const deployResult: SyncDeploymentResult = {
             name: flow.syncName,
             models: flow.models,
+            version: response.syncConfig.version as string,
             providerConfigKey: flow.providerConfigKey,
             type: flow.type
-        });
+        };
```

Comment:
you can have a one liner:
```
const deployResult: SyncDeploymentResult = {
...

...(response.syncConfig.version ? { version: response.syncConfig.version} : {})
}
```

Reply:
I personally find that less readable

Reply:
also agree on readability

Reply:
It is true the line is less readable but it allows a self-contained struct definition in a single block instead of having to mutate it in a second time. Not a massive deal since here it happens right below but as you know codebase evolves

Reply:
Yeah, understood your point